Dockerfile.ubuntu

```diff
@@ -326,9 +327,9 @@ WORKDIR /ovms
 # Test Coverage
 COPY ci/check_coverage.bat /ovms/
 ARG CHECK_COVERAGE=0
-COPY rununittest.sh prepare_llm_models.sh /ovms/
+COPY run_unit_tests.sh prepare_llm_models.sh install_va.sh /ovms/
```

Comment:
install_va.sh script is missing "-y" parameter is the last command: "apt install libva-dev". Right now this script can't be used in Dockerfile, since it is waiting for use input